src/frontend/src/catalog/system_catalog/rw_catalog/rw_actor_infos.rs

```diff
@@ -25,7 +25,6 @@ struct RwActorInfo {
     #[primary_key]
     actor_id: i32,
     fragment_id: i32,
-    node: JsonbVal,
```

Comment:
Not sure if this is relied somewhere (like, by Cloud). Shall we consider obtaining it from the fragment?